tests/basic/storage_pool_test.go

```diff
@@ -4923,32 +4937,35 @@ var _ = Describe("{ResizeClusterNoQuorum}", func() {
 				driverDownNodes = append(driverDownNodes, n)
 				i += 1
 			}
-		})
 
-		stepLog = fmt.Sprintf("Expanding pool on kvdb node using resize-disk")
-		Step(stepLog, func() {
-
-			log.FailOnError(err, fmt.Sprintf("Failed to get pool using UUID %s", selPool.Uuid))
-			expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+		})
 
-			log.InfoD("Current Size of the pool %s is %d", selPool.Uuid, poolToBeResized.TotalSize/units.GiB)
-			err = Inst().V.ExpandPool(selPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize, true)
-			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+		defer func() {
 			Step("set cluster to running", func() {
 				log.InfoD("set cluster to running")
 				for _, n := range driverDownNodes {
 					err := Inst().V.StartDriver(n)
 					log.FailOnError(err, "error starting driver on node %s", n.Name)
-					err = Inst().V.WaitDriverUpOnNode(n, 5*time.Minute)
+					err = Inst().V.WaitDriverUpOnNode(n, 10*time.Minute)
 					log.FailOnError(err, "error while waiting for driver up on node %s", n.Name)
 				}
 			})
 
-			isjournal, err := IsJournalEnabled()
-			log.FailOnError(err, "Failed to check if Journal enabled")
+		}()
+
+		stepLog = fmt.Sprintf("Expanding pool on non kvdb node using resize-disk")
+		Step(stepLog, func() {
+
+			log.FailOnError(err, fmt.Sprintf("Failed to get pool using UUID %s", selPool.Uuid))
+			expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+
+			log.InfoD("Current Size of the pool %s is %d", selPool.Uuid, poolToBeResized.TotalSize/units.GiB)
+			err = Inst().V.ExpandPool(selPool.Uuid, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, expectedSize, true)
+			dash.VerifyFatal(err != nil, true, "Pool expansion init failed?")
+
+			validErr := strings.Contains(err.Error(), "failed to find storage pool")
+			dash.VerifyFatal(validErr, true, fmt.Sprintf("vaery err is valid %s", err.Error()))
```

Comment:
Verify* err is valid. Does it skip valid error?

Reply:
yes

Reply:
Because of 

```
dash.VerifyFatal(err != nil, true, "Pool expansion init failed?")
```
 I do not think it will reach 

```
validErr := strings.Contains(err.Error(), "failed to find storage pool")
dash.VerifyFatal(validErr, true, fmt.Sprintf("vaery err is valid %s", err.Error()))
```

Reply:
we are expecting err not to be nil. if err !=nil it will execute next verification if not error it will fail at first verification only. that is the expected behavior

Reply:
My bad. Thank you.